webextensions/api/topSites.json

```
@@ -28,7 +28,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/topSites/MostVisitedURL",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "20"
```

:x: Same as `topSites`.

```suggestion
                "version_added": "19"
```